packages/angular_devkit/build_angular/src/angular-cli-files/utilities/bundle-calculator.ts

```diff
@@ -64,6 +64,7 @@ class InitialCalculator extends Calculator {
     const initialChunks = this.compilation.chunks.filter(chunk => chunk.isOnlyInitial());
     const size: number = initialChunks
       .reduce((files, chunk) => [...files, ...chunk.files], [])
+      .filter((file: string) => !/\.map$/.test(file))
```

Comment:
endsWith please.  Should be significantly faster and more obvious as to intent.

Reply:
updated